log4rs.yaml

```diff
@@ -62,6 +76,10 @@ loggers:
     level: info
     appenders:
       - rest_requests
+  app::cache:
```

Comment:
This should be `backend::cache` as it depends on the redis backend to be running. It should append to the `backend_requests` appender

Reply:
https://github.com/aetheric-oss/svc-cargo/pull/43/commits/5eccc2bc4b4335866a129e31c449d470990edf83